src/T2DMap.cpp

```diff
@@ -3119,18 +3119,18 @@ void T2DMap::slot_setCharacter()
                                                   QLineEdit::Normal,    // QLineEdit::EchoMode mode = QLineEdit::Normal
                                                   QString(),            // const QString & text = QString()
                                                   &isOk,                // bool * ok = 0
-                                                  0,                    // Qt::WindowFlags flags = 0
+                                                  Qt::Widget,           // Qt::WindowFlags flags = 0
```

Comment:
This confused me at first, until I checked `enum Qt::WindowType` and `flags Qt::WindowFlags` and found that `Qt::Widget` ___is___  the `enum` value corresponding to 0, of which is said:
> This is the default type for `QWidget`. Widgets of this type are child widgets if they have a parent, and independent windows if they have no parent. See also `Qt::Window` and `Qt::SubWindow`.